examples/block_jid.py

```diff
@@ -114,6 +116,11 @@ def jid(s):
             help="If given, prints the block list at the end of the operation",
         )
 
+        config_default = os.path.expanduser(
+            '~/.config/aioxmpp_credentials.ini')
```

Comment:
I see that you have a strong opinion towards something HOME-based. What is the reason for that? I don’t see one, but if you explain yours to me it might start to make sense to me. Currently, I do not see any advantage over simply defaulting to the examples directory (which is easy to detect, something along the lines of ``os.path.dirname(sys.argv[0])``).